jujugui/static/gui/src/app/components/user-profile/user-profile.js

```diff
@@ -198,7 +198,7 @@ YUI.add('user-profile', function() {
           model.owner = model.ownerTag;
           return model;
         });
-      } else {
+      } else if (data.map) {
```

Comment:
Is this intended as a type check? Could `else if (data && data.length) be sufficient here?